public/style.css

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
you could also implement this in a layout.erb view to be rendered on each of your views